components/shell/bash/patches/solaris-009.configure.patch

```diff
@@ -1,95 +1,32 @@
-# Solaris-specific ./configure stuff.
-# 1. We need to link bash with -lget.
-# 2. It's a release build, therefore we really don't want debugging
-# 3. We really want to use ncurses because bash is very compatible
-# with it.
-# 4. Don't try to use -static (OI addition)
---- configure.~1~	2014-02-11 19:38:00.000000000 +0400
-+++ configure	2016-03-03 17:10:01.047378168 +0300
-@@ -1,4 +1,4 @@
--#! /bin/sh
-+#!/bin/bash
- # From configure.ac for Bash 4.3, version 4.063.
- # Guess values for system-dependent variables and create Makefiles.
- # Generated by GNU Autoconf 2.69 for bash 4.3-release.
-@@ -15,6 +15,8 @@
- ## M4sh Initialization. ##
- ## -------------------- ##
- 
-+export LD_OPTIONS="-lc -lm -lgen -lsocket -lnsl -lresolv -lncurses -z ignore"
+--- configure.ac.orig	2017-05-08 03:23:15.044220076 +0200
```

Comment:
Alright, autoreconf cannot work because autoheader does not work.
I spent some time trying to fix it but apparently it is not meant to be used since most declarations are missing. I guess they just use the config.h as is. Even using aclocal breaks the build. See if the latest revision of the PR works for you.